google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddListBasedUserList.java

```diff
@@ -0,0 +1,152 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Creates a combination user list containing users that are present on any one of the provided user
+ * lists.
+ */
+public class AddListBasedUserList {
+
+  private static class AddListBasedUserListParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private Long customerId;
+
+    @Parameter(names = ArgumentNames.USER_LIST_ID, required = true)
+    private List<Long> userListIds;
+  }
+
+  public static void main(String[] args) {
+    AddListBasedUserListParams params = new AddListBasedUserListParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+      params.userListIds =
+          Arrays.asList(
+              Long.parseLong("INSERT_USER_LIST_ID_HERE"),
+              Long.parseLong("INSERT_USER_LIST_ID_HERE"));
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new AddListBasedUserList().runExample(googleAdsClient, params.customerId, params.userListIds);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @param userListIds the IDs of the lists to be used for the new combination user list.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(
+      GoogleAdsClient googleAdsClient, long customerId, List<Long> userListIds) {
+    // Creates the rule operand info targeting the provided list IDs.
```

Comment:
Not sure I understand this comment - could you describe why this is being done rather than what is being done?